node/cmd/guardiand/node.go

```diff
@@ -1577,9 +1585,15 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	reliableTss, err := tss.NewReliableTSS(tssGuardianStorage)
```

Comment:
consider matching the variable name (reliableTSS) to what the reader is expected to understand from the log (i.e., that it's the tss engine)